src/widgets/CategoryWidget.js

```diff
@@ -94,12 +92,16 @@ function CategoryWidget(props) {
   );
 
   return (
-    <WrapperWidgetUI title={props.title} loading={loading} {...props.wrapperProps}>
+    <WrapperWidgetUI
+      title={props.title}
+      loading={widgetLoaders[props.id]}
+      {...props.wrapperProps}
+    >
       <CategoryWidgetUI
         data={categoryData}
         formatter={props.formatter}
         labels={props.labels}
-        loading={loading}
+        loading={widgetLoaders[props.id]}
```

Comment:
I would use `isLoading` for the prop name

Reply:
Indeed, seems better